scripts/conversion_script.py

```diff
@@ -583,9 +583,12 @@ def update_insights_inventory():
     output, returncode = run_subprocess(cmd=["/usr/bin/insights-client"])
 
     if returncode:
-        raise ProcessError(
-            message="Failed to update Insights Inventory by registering the system again.",
-            report="insights-client execution exited with code '%s' and output:\n%s"
+        # FIXME: We can't raise ProcessError here because we call it in finally
```

Comment:
I think we can do differently here... Maybe move the `update_insights_inventory` up to the try block? It seems pretty important to me to tell the user that something went wrong with that, other than just printing to the stdout, which might get discard.

Reply:
Yeah sure, I just created this quickly to mark the problem :) But what if the conversion was success but the upload failed? I will try to modify the message to reflect that :thinking:

Reply:
Alright moved it at the end of `try` before the success log print, since we call it only if conversion suceeded (no ProcessError raised - rollback or exit 1).

I also changed the message to include `'Conversion succeeded but update of Insights Inventory by registering the system again failed.'` I feel like that is fair, little questionable is if we should keep alert for UI in this case, maybe better to leave it?